pkg/fleetautoscalers/fleetautoscalers_test.go

```diff
@@ -1171,7 +1171,7 @@ func TestApplyCounterPolicy(t *testing.T) {
 				BufferSize:  intstr.FromInt(10),
 			},
 			want: expected{
-				replicas: 14,
+				replicas: 15,
```

Comment:
This is fine by me. It's really just a question of if someone has a MaxCapacity that's not evenly divisible by the Counter or List Capacity do we want to slightly undershoot the MaxCapacity or slightly overshoot it. I imagine in most cases people will pick evenly divisible numbers.

Reply:
Yeah you are right. But there are some places in the code where we use Ceil. Due to this I saw some flapiness in auto scaling events. 

Example : If the calculated value was say 3.4 the desired capacity will be first set to 3 and then it will be immediately set to 4 in the next run. Isn't it better to have this consistent?